src/extension/extensionServer.ts

```diff
@@ -235,6 +239,18 @@ export class ExtensionServer implements vscode.Disposable {
                 TargetPlatformHelper.checkTargetPlatformSupport(mobilePlatformOptions.platform);
                 return mobilePlatform.beforeStartPackager()
                     .then(() => {
+                        generator.step("getReactNativeVersion");
+                        return ReactNativeProjectHelper.getReactNativePackageVersionFromNodeModules(
+                            path.resolve(request.arguments.cwd, "node_modules", "react-native")
+                            );
+                    })
+                    .catch(err => {
+                        const noReactNativePackageError = ErrorHelper.getInternalError(InternalErrorCode.ReactNativePackageIsNotInstalled);
```

Comment:
[Code duplicating](https://github.com/microsoft/vscode-react-native/pull/1093/files#diff-fedcb4bc4e0aeb026fbb6f9752cda410R414)

Reply:
I moved a part of this code to the `getReactNativePackageVersionFromNodeModules` function.